ur/package.xml

```diff
@@ -5,6 +5,7 @@
   <version>2.4.9</version>
   <description>Metapackage for universal robots</description>
   <maintainer email="exner@fzi.de">Felix Exner</maintainer>
+  <maintainer email="dipentima@fzi.de">Vincenzo Di Pentima</maintainer>
   <maintainer email="wilbrandt@fzi.de">Robert Wilbrandt</maintainer>
```

Comment:
Maybe it makes sense to remove Robert here in the same run. The package itself doesn't contain anything so it doesn't make sense to keep him as an author there, in my opinion.

Reply:
I was of the same opinion, but I was not sure it was the right thing to do. I will do so :+1: